compiler/cli_reporter.nim

```diff
@@ -0,0 +1,3405 @@
+import reports,
```

Comment:
Need at least a summary of what this module does.

Reply:
provided in module itself